omrse-base.obo

```diff
@@ -4210,6 +4210,7 @@ is_a: BFO:0000038 ! one-dimensional temporal region
 [Term]
 id: OMRSE:00000085
 name: obsolete socio-legal human social role
+is_a: ObsoleteClass ! Obsolete Class
 is_a: OMRSE:00000001 ! human social role
```

Comment:
This should have been removed, and it was in the other files. Not sure what the problem is.